hw-latest/fpga/FPGA_Instructions.md

```diff
@@ -0,0 +1,91 @@
+_*SPDX-License-Identifier: Apache-2.0<BR>
+<BR>
+<BR>
+Licensed under the Apache License, Version 2.0 (the "License");<BR>
+you may not use this file except in compliance with the License.<BR>
+You may obtain a copy of the License at<BR>
+<BR>
+http://www.apache.org/licenses/LICENSE-2.0 <BR>
+<BR>
+Unless required by applicable law or agreed to in writing, software<BR>
+distributed under the License is distributed on an "AS IS" BASIS,<BR>
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.<BR>
+See the License for the specific language governing permissions and<BR>
+limitations under the License.*_<BR>
+
+# **Caliptra FPGA Guide** #
+
+### Requirements: ###
+ - Vivado
+   - `Version v2022.2`
+ - FPGA
+   - `ZCU104 Development Board`
+   - https://www.xilinx.com/products/boards-and-kits/zcu104.html
+ - [Not fully enabled] JTAG debugger
+   - `Olimex ARM-USB-TINY-H`
+
+### ZCU104 Processing System One Time Setup: ###
+1. Install ZCU104 SD card image
+   - https://ubuntu.com/download/amd-xilinx
+1. Configure SW6 to boot from SD1.
+   - Mode SW6[4:1]: OFF, OFF, OFF, ON
```

Comment:
Can we include this picture?

![PXL_20230419_183359704](https://github.com/chipsalliance/caliptra-sw/assets/993854/3bd853f7-ef74-481f-9db1-947eda0caafc)

Reply:
Including picture in doc.